docs/docs/querying.md

```diff
@@ -287,11 +288,18 @@ something.findOne({
     // Will order by name on an associated User
     [User, 'name', 'DESC'],
 
+    // Will order by name on an associated User using the attribute's association alias
+    ['User.name', 'DESC'],
```

Comment:
The whole idea of this is to allow a JSON object to be passed into without reference to the models themselves.  For example, before the literal was removed from ordering you could reference the column you want to sort without having to wrap the object in a Sequelize method.  This also adds the functionality of being able to pass the alias of an attribute to the order instead of the field name.  This functionality also gives the developer the ability to pass JSON straight from an HTTP request in the order property without having to do sanitation or manipulation.

An old example may be would look something like,
```javascript
order: [
  ['"Task.Project"."created_at"', 'DESC']
]
```
but this now prefixes the model name to the order clause `"Subtask"."Task.Project"."created_at" DESC`

This feature not only fixes that but brings back that same functionality in a cleaner and more effective manner.